diff -Nur autolatex-5.1/Makefile.PL autolatex-5.1.new/Makefile.PL
--- autolatex-5.1/Makefile.PL	2007-09-23 07:22:06.000000000 +0200
+++ autolatex-5.1.new/Makefile.PL	2007-09-23 07:23:28.000000000 +0200
@@ -339,6 +339,10 @@
 		installLink($cfgFile,$INSTALLDIRECTORY);
 		$cfgFile = installCfg('default.cfg','config');
 		installLink($cfgFile,$INSTALLDIRECTORY,'default.cfg');
+		# Remove the config file because it will be re-generated
+		# by DebConf scripts from a configuration model
+		unlink("$cfgFile");
+		install('default.cfg',$INSTALLDIRECTORY,'default.cfg.model');
 	}
 	else {
 		install('default.ist',$INSTALLDIRECTORY);
